spec.bs

```diff
@@ -934,6 +983,30 @@ To <dfn>validate and convert auction ad config</dfn> given an {{AuctionAdConfig}
       1. Set |auctionConfig|'s [=auction config/seller signals=] to the result of
         [=serializing a JavaScript value to a JSON string=], given |result|.
     * To handle an error, set |auctionConfig|'s [=auction config/seller signals=] to failure.
+1. If |config|["{{AuctionAdConfig/auctionNonce}}"] [=map/exists=]:
+  1. Set |auctionConfig|'s [=auction config/auction nonce=] to the result of running

```

Comment:
nit: Note that not every `if` body needs to be in a nested step. In fact, keeping it all on one line would probably be better for readability.